pvr.iptvsimple/resources/language/resource.language.en_gb/strings.po

```diff
@@ -705,3 +715,13 @@ msgstr ""
 msgctxt "#30724"
 msgid "If a stream_mode property is not specified along with an inputstream.ffmpegdirect property always add it for any supported stream types. Note that this setting has no effect on catchup VOD streams that are not live."
 msgstr ""
+
+#. help: Timeshift - timeshiftEnabledAll
+msgctxt "#30725"
+msgid "Enable the timeshift feature for all streams. Will only apply to streams that do not select a specific inputstream addon for playback. The `inputstream.ffmpegdirect` addon will be used for timeshift."
+msgstr ""
+
+#. help: Timeshift - timeshiftEnabledUdp
+msgctxt "#30726"
+msgid "Enable the timeshift feature for UDP based streams. Will only apply to streams that do not select a specific inputstream addon for playback. The `inputstream.ffmpegdirect` addon will be used for timeshift."
```

Comment:
The <b><c>\`</c></b> not works within Kodi, maybe you can use `[B]` or `[I]`.

Would be a good feature in Kodi to allow also a `[C]...[\C]` and to use e.g. Courier font for it.

Reply:
Ah, yes, whoops. Copy paste mistake.